docs/src/references/c8y-configuration-management.md

```diff
@@ -64,8 +91,38 @@ $ ls -l /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 -rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 ```
 
+The configuration plugin can also act as configuration proxy for child-devices.
+For that to work for a child device named `$CHILD_DEVICE_ID`,
+two files must be added under `/etc/tedge/operations/c8y/$CHILD_DEVICE_ID`
+in order to declare the associated capabilities to Cumulocity.
+
+These two files can be created using the `c8y_configuration_plugin --init` option and providing child names:
+
+```shell
+$ sudo c8y_configuration_plugin --init child-1 child-2
+
+$ ls -l /etc/tedge/operations/c8y/child-1/c8y_UploadConfigFile
+-rw-r--r-- 1 tedge tedge 95 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_UploadConfigFile
+  
+$ ls -l /etc/tedge/operations/c8y/child-1/c8y_DownloadConfigFile
+-rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_DownloadConfigFile
+```
+
```

Comment:
Question, does this `--init child-1 child-2` also create toml files for them in `etc/tedge/c8y/$CHILD_DEVICE_ID/c8y-configuration-plugin.toml`?

Reply:
No. The `c8y-configuration-plugin.toml` files are to be provided by the child-devices themselves.

Reply:
The steps how the thin-edge device will get the `c8y-configuration-plugin.toml` from the child device need to be clearly documented. It's the missing part in our prototype implementation. File transfer? MQTT message from child-device and c8y_configuration_plugin converts it to a file as proposed by @cstoidner [here](https://github.com/thin-edge/thin-edge.io/pull/1236/files?short_path=0d43010#diff-0d430102a9a7f4e4431390a85806a990b51d2143dbc97bf596a267c893e1f693)?

Reply:
Indeed, this is a critical missing point. I still need to describe it. However, the idea I have in mind differs from what is proposed by Christoph. Why use MQTT for this configuration file when the child devices send all the other configuration files over HTTP? Why use JSON for this configuration file over the network while it's stored as TOML?

Furthermore, a key design idea for the `c8y_configuration_plugin` is to have this plugin freely configured from the cloud using the same mechanism as for all the other configuration files. We need to follow the same pattern here, i.e. the `c8y-configuration-plugin.toml` for a child device must be configurable from the cloud. So the content for this file will flow as the others from the cloud to the child-device via the main thin-edge device. In this very specific case, the configuration plugin will not only act as a proxy, it will also take a copy of this configuration for the child device. 

What is not clear yet for me, is when the very first version is uploaded from the child device to the main thin-edge device.
This can be done on request from by the main device or at start of the child device

Reply:
> The steps how the thin-edge device will get the c8y-configuration-plugin.toml from the child device need to be clearly documented. It's the missing part in our prototype implementation. 

Here is my proposal https://github.com/thin-edge/thin-edge.io/pull/1417/commits/a4bb877926ea7e0ae931f4c63b6a9bf9c27bb645